pvlib/modelchain.py

```diff
@@ -867,30 +867,12 @@ def spectral_model(self, model):
                 self._spectral_model = self.no_spectral_loss
             else:
                 raise ValueError(model + ' is not a valid spectral loss model')
-        else:
+        elif model is None:
+            # not setting a model is equivalent to setting no_loss
+            self._spectral_model = self.no_spectral_loss
```

Comment:
`model=None` isn't currently documented.  We should document that `None` is equivalent to `no_loss`, or remove it.  I lean towards the former.